invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -107,6 +107,7 @@ pre.export.result {
 
 .rdm-logo {
   width: 250px;
+  max-height: @relative75px;
```

Comment:
```suggestion
  max-height: @75px;
```
could you include screenshots on how the logo looks like when different sizes of images are added as a logo? for example images with a big disproportion between width and height (very high or very wide)